### DISCUSSION TITLE
Supported transformers models?

### DISCUSSION THREAD

**Discussion by jammiguelyfish** [guidance-ai/guidance#977] [Q&A]
Is it safe to say that all models in transformers are supported?

**Comment by Harsha-Nori** [comment]
Hi @jammiguelyfish , we try our best to run on all transformers models and wrote the guidance code to be as general as we could! Of course we can't test all of them, and bugs do appear, so please let us know if any models don't run and we're happy to take a look. One key call out is that we currently don't support multimodal models, but hope to have something to say on that front soon!
